src/status.hpp

```diff
@@ -202,6 +202,9 @@ enum class StatusCode {
     NODE_LIBRARY_LOAD_FAILED_SYM,
     NODE_LIBRARY_MISSING,
 
+    // Model control API
+    OK_RELOAD_NOT_NEEDED, /*!< Operation succeeded but no config reload was needed */
```

Comment:
Can we change name to make it clear that config reload was not required?
Not to confuse with model reload (due to changes in versions or dynamic shape/batch size)